app/views/main.html

```diff
@@ -0,0 +1,23 @@
+<h2>Awesome Todo-List</h2>
+<div class="container">
+
+  <form role="form" ng-submit="addTodo()">
```

Comment:
calls to `addTodo()` create a dependency on the scope having a method `addTodo()`.  There are ways to handle this in Angular using directives.  It's more advanced, but worth investigating if you want to understand Angular a bit deeper.